src/interface/outbound/api_template/api-outbound-template.yaml

```diff
@@ -1,38 +1,36 @@
+# Note: I've manually edited the `responses` section to replace all errors with a single
```

Comment:
Oh nice, did not know that. I'll make sure to update the snippets with that another time.